Assets/Plugins/Android/Editor/PlatformSpecificConfigEditorAndroid.cs

```diff
@@ -28,34 +28,11 @@
 namespace PlayEveryWare.EpicOnlineServices
 {
 
-    public class PlatformSpecificConfigEditorAndroid : IPlatformSpecificConfigEditor
+    public class ConfigSectionAndroid : ConfigSection<EOSAndroidConfig>
     {
-        public static string ConfigFilename = "eos_android_config.json";
-        EOSConfigFile<EOSAndroidConfig> configFile;
+        public ConfigSectionAndroid() : base("Android", "eos_android_config.json") { }
 
-        [InitializeOnLoadMethod]
-        static void Register()
-        {
-            EpicOnlineServicesConfigEditor.AddPlatformSpecificConfigEditor(new PlatformSpecificConfigEditorAndroid());
-        }
-
-        public string GetNameForMenu()
-        {
-            return "Android";
-        }
-
-        public void Awake()
-        {
-            var configFilenamePath = EpicOnlineServicesConfigEditor.GetConfigPath(ConfigFilename);
-            configFile = new EOSConfigFile<EOSAndroidConfig>(configFilenamePath);
-        }
-
-        public void LoadConfigFromDisk()
-        {
-            configFile.LoadConfigFromDisk();
-        }
```

Comment:
These are all examples of functionality that was moved to a base class ("ConfigSection").